nixos/modules/services/networking/centrifugo.nix

```diff
@@ -0,0 +1,81 @@
+{ config, lib, pkgs, ... }:
+let
+  cfg = config.services.centrifugo;
+
+  format = pkgs.formats.json { };
+
+  configFile = format.generate "centrifugo.json" cfg.settings;
+in
+{
+  options.services.centrifugo = {
+    enable = lib.mkEnableOption (lib.mdDoc "Centrifugo messaging server");
+
+    package = lib.mkPackageOptionMD pkgs "centrifugo" { };
+
+    settings = lib.mkOption {
+      default = { };
+      type = format.type;
+      description = lib.mdDoc ''
+        Declarative Centrifugo configuration. See the [Centrifugo
+        documentation] for a list of options.
```

Comment:
I have never seen this syntax for a link and it is not listed in the commonmark [website](https://commonmark.org/help/)

Reply:
See https://spec.commonmark.org/0.30/#link-reference-definitions

Reply:
This makes reading raw Markdown a bit easier since long URLs are not interleaved with text.